Source/Spray/SprayInjection.H

```diff
@@ -7,6 +7,249 @@
  problem-specific SprayParticlesInitInsert.cpp file, when necessary.
 */
 
+std::ifstream&
+GotoLine(std::ifstream& file, long int num)
+{
+  file.seekg(std::ios::beg);
+  for (int i = 0; i < num - 1; ++i) {
+    file.ignore(std::numeric_limits<std::streamsize>::max(), '\n');
+  }
+  return file;
+}
+
+void
+SprayParticleContainer::sprayInjectionfromDPMFile(
+  const amrex::Real time,
+  SprayJet* spray_jet,
+  const amrex::Real sim_dt,
+  const int level)
+{
+
+  if (!spray_jet->jet_active(time) || sim_dt <= 0.) {
+    amrex::Print() << "Warning! Spray jet " << spray_jet->jet_name()
```

Comment:
Try to wrap terminal output with `if (m_verbose > 0) {}` for classes that have a verbosity parameter so users can turn off diagnostic output if it is undesired.